### ISSUE TITLE
Fatal error when creating Kubernetes cluster with hosts using encrypted ssh keys

### ISSUE THREAD

**Issue by soumyalj** [rancher/rke#18]
rke version:  0.0.1-dev
1. Create encrypted ssh keys using ssh-keygen 
2. Add these keys to .ssh/authorized_keys file on the hosts that are used to create the cluster(specified in cluster.yml)
3. Create cluster using `./rke cluster up` command.
Cluster creation fails with a fatal error.
```
rke soumya$ ./rke_darwin-amd64 cluster up --cluster-file cluster.yml 
INFO[0000] Building Kubernetes cluster                  
INFO[0000] [ssh] Start tunnel for host [soumyarketest-01] 
INFO[0000] [ssh] Start tunnel for host [soumyarketest-01] 
INFO[0000] [ssh] Start tunnel for host [soumyarketest-02] 
WARN[0000] Failed to initiate new Kubernetes Client: stat admin.config: no such file or directory 
INFO[0000] [certificates] Generating kubernetes certificates 
INFO[0000] [certificates] Generating CA kubernetes certificates 
INFO[0000] [certificates] Generating Kubernetes API server certificates 
INFO[0000] [certificates] Generating Kube Controller certificates 
INFO[0000] [certificates] Generating Kube Scheduler certificates 
INFO[0001] [certificates] Generating Kube Proxy certificates 
INFO[0001] [certificates] Generating Node certificate   
INFO[0002] [certificates] Generating admin certificates and kubeconfig 
INFO[0002] [certificates] Deploying kubernetes certificates to Cluster nodes 
FATA[0002] Error configuring SSH: ssh: cannot decode encrypted private keys 
```

**Comment by galal-hussein** [comment]
you can test this fix using v0.0.2-dev release

**Comment by soumyalj** [comment]
Tested with rke version v0.0.2-dev. Creation of Kubernetes cluster when hosts are using encrypted ssh keys are successful. There is a password prompt as the keys are encrypted.
